cluster/centos/util.sh

```diff
@@ -88,26 +88,26 @@ function trap-add {
   local signal="${2-EXIT}"
   local cur
 
-  cur="$(eval "sh -c 'echo \$3' -- $(trap -p ${signal})")"
+  cur="$(eval "sh -c 'echo \$3' -- $(trap -p "${signal}")")"
   if [[ -n "${cur}" ]]; then
     handler="${cur}; ${handler}"
   fi
 
-  trap "${handler}" ${signal}
+  # shellcheck disable=SC2064 # Controlled expansion doesn't matter here
```

Comment:
s/Controlled expansion doesn't matter here/early expansion is intentional here/

That said, this is a bit of a footgun pattern, because any variables referenced by $1 inline will be expanded early too, but I don't think I see a better option here.